### PR TITLE
updated publication and blog pages to work with get_page_authors tag

### LINKED ISSUE #7333: Author images not showing on blog pages
### Describe the bug
We have gotten reports that author images are not appearing for blog posts on both mozfest and the mofo blog.

### PR DESCRIPTION
**Pull request by danielfmiranda**
Closes #7333 

**Link to sample test page**:

Test **Blog** Page with Author Images: https://foundation-s-7333-autho-4onovn.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/
Test **Publication** Page with Author Images: https://foundation-s-7333-autho-4onovn.herokuapp.com/en/publication-page-with-child-article-pages/

**Steps To Test**:

1. Visit the two pages above
2. Ensure that they both are displaying authors with and without images correctly. 
3. If they are, testing is complete! 

## Checklist

**Changes in Models:**
- [x] [Are my changes backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/master/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations)

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templatetags/homepage_tags.py [hunk 1]
```diff
@@ -32,17 +32,16 @@ def homepage_image_with_class(context, path, classname):
 
 
 @register.simple_tag
-def get_blog_page_authors(blog_page):
+def get_page_authors(parent_page):
```
Comment (danielfmiranda) [inline network-api/networkapi/wagtailpages/templatetags/homepage_tags.py:35]: It appears that this function already existed to return the authors of a blog page, I have updated it to accept both blog pages and publication pages, that way we can continue to use the single "author-images" template by calling this tag in both the blog page template and publication page template

Comment (Pomax) [inline network-api/networkapi/wagtailpages/templatetags/homepage_tags.py:35]: let's not call this "parent_page" though, this is still just `page`
```suggestion
def get_page_authors(page):
```

File: network-api/networkapi/wagtailpages/templates/wagtailpages/publication_page.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 {% block content%}
 
-  {% include "./fragments/publication_hero.html" with download_file=page.publication_file is_publication_page=page.is_publication_page %}
+  {% include "./fragments/publication_hero.html" with  parent_page=page download_file=page.publication_file is_publication_page=page.is_publication_page %}
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/publication_page.html:6]:
```suggestion
  {% include "./fragments/publication_hero.html" with  page=page download_file=page.publication_file is_publication_page=page.is_publication_page %}
```

File: network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/publication_hero.html [hunk 3]
```diff
@@ -1,4 +1,7 @@
-{% load i18n wagtailimages_tags %}
+{% load i18n wagtailimages_tags homepage_tags%}
+
+{% get_page_authors parent_page as authors %}
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/publication_hero.html:3]:
```suggestion
{% get_page_authors page as authors %}
```

**Review by Pomax — CHANGES_REQUESTED** [review]

**Comment by danielfmiranda** [comment]
Hi @Pomax! Thanks for your feedback, I have added the suggested changes, to update "parent_page" to "page" and have re-requested review. Thanks!

**Review by Pomax — APPROVED** [review]

**Pull request merged**